src/api/contract/errors.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+
+  ERR_MAKERTOKEN_ADDRESS_INVALID: `
```

Comment:
@prdn / @plutoegg imported the error codes and descriptions here I thought it could be useful to yield user-friendly messages. thoughts?

Reply:
I think definitely a good approach to transform the error codes into user friendly ones!